src/Catalog/Helpers/FeedHelpers.cs

```diff
@@ -144,155 +141,11 @@ public static async Task<IList<FeedPackageDetails>> GetPackages(HttpClient clien
                     packageId,
                     packageNormalizedVersion,
                     licenseNames,
-                    licenseReportUrl));
+                    licenseReportUrl,
+                    deprecationInfo: null));
```

Comment:
Does anything still use this? We should clean up all the V2 feed code eventually--maybe in a separate PR?

Reply:
Still used by `package2catalog` job.

Reply:
`package2catalog` will be broken by this work...we should probably remove it entirely

alternatively, we could update it to use the DB, but I don't think the job has been used in many years so it's safe to just delete

Reply:
@joelverhagen @skofman1 anyone remember the purpose of `package2catalog`? Should we keep maintaining it? Or can I go ahead and remove it completely?

And should it be part of this PR or tracked by separate issue? :)

Reply:
Talked to @joelverhagen offline (@skofman1 is on vacation this week and next). We came to the consensus that it should be removed, along with all the code that is only remaining because of it. I would do it in a separate PR though.

Reply:
Tracking issue: https://github.com/nuget/engineering/issues/2504